src/main/java/eod/effect/Heal.java

```diff
@@ -14,7 +14,7 @@ public Heal(Player player, int hp) {
         this.player = player;
     }
 
-    public void from(WarObject[] objects) {
+    public void to(WarObject[] objects) {
```

Comment:
// tOdO: add to(WarObject[] objects, int num) to heal multiple characters.

Reply:
Added